src/audio/Kconfig

```diff
@@ -32,6 +32,16 @@ config HOST_DMA_RELOAD_THRESHOLD
 	  The DMA buffer threshold in milliseconds to trigger host DMA
 	  reloading.
 
+config HOST_DMA_STREAM_SYNCHRONIZATION
+	bool "Stream DMA Transfers Synchronization"
+	default y if ACE
+	default n
+	help
+	  Enable synchronized FPI updates of HD-A gateways belonging to a group defined by the
```

Comment:
Maybe worth putting "Buffer Position Increment (FPI)" here once explicitly

Reply:
> is this "FPI" - the same?

Yes.

> But shouldn't it be BPI or BFPI

**BPI** No, **BFPI** more likely

**DGBFPI** stands for **D**SP **G**ateway **B**uffer **F**irmware **P**ointer **I**ncrement. Which is probably the generic name for the registers of Link Input, Link Output, Host Input and Host Output.

This three-letter acronym (FPI) appears in many places in the documentation. I think just expanding the acronym will be enough.